includes/admin/auth-service-actions.php

```diff
@@ -43,7 +43,7 @@ public function __construct()
 		//add_action('wp_ajax_nopriv_oauth_deauthenticate_site', [$this, 'oauth_deauthenticate_site']);
 		// Do toke check here.
 		$post_type = get_post_type();
-		if($post_type = 'calendar'){
+		if($post_type == 'calendar'){
```

Comment:
@Akhill2020 The previous condition was wrong and even this condition is not working properly. What should be it?